project/SbtMultiJvm.scala

```diff
@@ -167,10 +167,11 @@ object MultiJvmPlugin extends AutoPlugin {
       // the first class wins just like a classpath
       // just concatenate conflicting text files
       assembly / assemblyMergeStrategy := {
-        case n if n.endsWith(".class") => MergeStrategy.first
-        case n if n.endsWith(".txt")   => MergeStrategy.concat
-        case n if n.endsWith("NOTICE") => MergeStrategy.concat
-        case n                         => (assembly / assemblyMergeStrategy).value.apply(n)
+        case n if n.endsWith(".class")  => MergeStrategy.first
+        case n if n.endsWith(".txt")    => MergeStrategy.concat
+        case n if n.endsWith("NOTICE")  => MergeStrategy.concat
+        case n if n.endsWith("LICENSE") => MergeStrategy.concat
```

Comment:
I am wondering why this is an issue with this PR specifically and not with Pekko in general?

Reply:
It need a [SbtAssembly](https://github.com/sbt/sbt-assembly) to package the jar. and Netty 4 has those files in every little netty-* jars